engine/src/main/java/org/terasology/rendering/world/viewDistance/ViewDistance.java

```diff
@@ -13,14 +13,14 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package org.terasology.rendering.world;
+package org.terasology.rendering.world.viewDistance;
```

Comment:
Hmm, this probably should be called WorldViewDistance or something for that matter. *shrug*